DSharpPlus/Entities/BaseDiscordMessageBuilder.cs

```diff
@@ -44,7 +45,7 @@ public T SuppressNotifications()
     /// <summary>
     /// Gets or sets a poll for this message.
     /// </summary>
-    public DiscordPoll? Poll { get; set; }
+    public PollCreatePayload? Poll { get; set; }
```

Comment:
can we take a DiscordPollBuilder here instead? i'm not a fan of exposing the REST payload type

Reply:
or a DiscordPoll, for that matter